src/main/java/io/github/redouane59/twitter/dto/tweet/TweetV2.java

```diff
@@ -221,7 +239,8 @@ public TweetType getTweetType() {
     private String                   inReplyToUserId;
     @JsonProperty("referenced_tweets")
     private List<ReferencedTweetDTO> referencedTweets;
-    private JsonNode                 entities;
+    @JsonProperty("entities")
```

Comment:
this line is useless no ?

Reply:
As long as we do not rename the class attribute `entities`, the annotation can be removed :)